src/metadatas.rs

```diff
@@ -144,6 +152,35 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
     }
 }
 
+impl Metadata {
+    pub fn enrich_with_advanced_infos(&mut self, gtfs: &gtfs_structures::Gtfs) {
+        self.stops_with_wheelchair_info_count = Some(stops_with_wheelchair_info_count(gtfs));
+    }
+}
+
+fn stops_with_wheelchair_info_count(gtfs: &gtfs_structures::Gtfs) -> usize {
+    gtfs.stops
+        .iter()
+        .filter(|(_, stop)| {
+            if stop.wheelchair_boarding != gtfs_structures::Availability::InformationNotAvailable {
+                // information is present
+                return true;
+            } else {
+                // information not present, check for inheritance from parent station
+                let parent_wheelchair_boarding = stop
+                    .parent_station
+                    .as_ref()
+                    .and_then(|parent| gtfs.stops.get(parent))
+                    .and_then(|s| Some(s.wheelchair_boarding));
```

Comment:
really minor, but I think that here you can just do:
```rust
.map(|s| s.wheelchair_boarding);
```

Reply:
looks better, thanks!